gpa.py

```diff
@@ -103,13 +93,43 @@ def get_semesters(course_records):
    -------
    array of semesters
    """
-   semesters=set()
+   semesters = set()
    for d in (course_records['courses']):
       for key, value in d.items():
-         if key=='term':
+         if key == 'term':
                semesters.add(value) 
```

Comment:
Replace this entire block with 
`semesters = set([course['term'] for course in course_records['courses']])`

Reply:
DOne